server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -59,33 +59,42 @@ export class TenantManager implements core.ITenantManager {
 		documentId: string,
 		includeDisabledTenant = false,
 	): Promise<core.ITenant> {
-		const restWrapper = new BasicRestWrapper();
 		const [details, gitManager] = await Promise.all([
-			restWrapper.get<core.ITenantConfig>(`${this.endpoint}/api/tenants/${tenantId}`, {
-				includeDisabledTenant,
-			}),
-			this.getTenantGitManager(tenantId, documentId, includeDisabledTenant),
+			this.getTenantConfig(tenantId, includeDisabledTenant),
+			this.getContainerGitManager(tenantId, documentId, undefined, includeDisabledTenant),
 		]);
 
 		const tenant = new Tenant(details, gitManager);
 
 		return tenant;
 	}
 
-	public async getTenantGitManager(
+	public async getTenantConfig(
+		tenantId: string,
+		includeDisabledTenant = false,
+	): Promise<core.ITenantConfig> {
```

Comment:
I see the `ITenantConfig` is listed as `private` for `ITenant`. Not sure why we design like that. Let me know if you have any concerns for me to expose this api

Reply:
Odds are it was just inline code because it wasn't needed anywhere else, so no point in exposing it. This looks fine to me

Reply:
It is because customer data is a key to many dictionary:
![image](https://user-images.githubusercontent.com/3719246/233373038-73af8ebc-d50a-4b6e-9d81-dcc0ed513de8.png)

I will cast it to string then

Reply:
Move it to private given we are not using in latest design